source/agora/node/Validator.d

```diff
@@ -449,6 +449,21 @@ public class Validator : FullNode, API
         return new AdminInterface(this.config.validator.key_pair, this.clock);
     }
 
+    /// Ditto
+    public override UTXO getValidatorUTXO (in PublicKey pubkey) @trusted nothrow
+    {
+        auto hash = this.enroll_man.validator_set.getEnrolledUTXO(
+                this.ledger.getBlockHeight(), pubkey);
+        if (hash != Hash.init)
+        {
+            UTXO utxo;
+            this.utxo_set.peekUTXO(hash, utxo);
```

Comment:
peekUTXO has a bool return value to indicate if the peeking was successful

Reply:
What do you mean? I used the function to get a UTXO for the hash.

Reply:
> What do you mean? I used the function to get a UTXO for the hash.

what if 
this.utxo_set.peekUTXO(hash, utxo); 
returns false? it would mean 
utxo == UTXO.init and we would return that

Reply:
You mean the line of `if (hash != Hash.init)` is not needed?

Reply:
I will check the returned value with assert. Ty.